Doc/library/string.rst

```diff
@@ -704,10 +718,20 @@ Replacing ``%x`` and ``%o`` and converting the value to different bases::
    >>> "int: {0:d};  hex: {0:#x};  oct: {0:#o};  bin: {0:#b}".format(42)
    'int: 42;  hex: 0x2a;  oct: 0o52;  bin: 0b101010'
 
-Using the comma as a thousands separator::
+Using the comma or the underscore as a digit group separator::
 
    >>> '{:,}'.format(1234567890)
    '1,234,567,890'
+   >>> '{:_}'.format(1234567890)
+   '1_234_567_890'
+   >>> '{:_b}'.format(1234567890)
+   '100_1001_1001_0110_0000_0010_1101_0010'
+   >>> '{:_x}'.format(1234567890)
+   '4996_02d2'
+   >>> '{:,}'.format(123456789.123456789)
+   '123,456,789.12345679'
+   >>> '{:_}'.format(123456789.123456789)
+   '123_456_789.12345679'
```

Comment:
This is not from backported code.  Last example from f39a07be47, other - new.

Reply:
Indeed, I added extra comma example in the very last commit. I am going to add this one and additional w/ fractional part to main. The reason I added an additional example is that currently there is 1 example with comma and 4 with underscore and in main the ratio is even worse.

However, if consistency between PRs is required, I can remove this example from here. In that case, I suggest removing both floating-point examples from this PR.

Reply:
I removed both floating point examples. They do not give that much new information anyway unlike in 3.14.